custom_components/smartthinq_sensors/wideq/devices/dehumidifier.py

```diff
@@ -314,9 +319,36 @@ def water_tank_full(self):
             return None
         return self._update_feature(DehumidifierFeatures.WATER_TANK_FULL, value)
 
+    @property
+    def notification_light(self) -> bool | None:
+        """Return notification light status."""
+        try:
+            key = self._get_state_key(STATE_NOTIFICATION_LIGHT)
+        except:
+            key = None
+            _LOGGER.exception("LGE ThinQ dehumidifier - unable to get Notification Light status")
+        if key is None:
+            ntf_real_value = None
+            ntf_light_int_value = None
+            ntf_light_bool_val = None
+        else:
+            ntf_real_value = self.lookup_range(key)
+            ntf_light_int_value = self.to_int_or_none(ntf_real_value)
+            if ntf_light_int_value is None:
+                ntf_light_bool_val = None
+                _LOGGER.warning(f"LGE ThinQ dehumidifier Notification light is {ntf_real_value}. int {ntf_light_int_value}. bool {ntf_light_bool_val}")
+            elif ntf_light_int_value > 0:
+                ntf_light_bool_val = True
+            else:
+                ntf_light_bool_val = False
+            if ntf_light_bool_val is not None:
+                _LOGGER.debug(f"LGE ThinQ dehumidifier Notification light is {ntf_real_value}. int {ntf_light_int_value}. bool {ntf_light_bool_val}")
+        return self._update_feature(DehumidifierFeatures.NOTIFICATION_LIGHT, ntf_light_bool_val)
```

Comment:
At this point you can just simplify with 

```
return self._update_feature(DehumidifierFeatures.NOTIFICATION_LIGHT, ntf_light_int_value > 0, False)
``` 
But are you sure that this is an int with value > 0 to define `light on`?